src/resources/views/fields/select.blade.php

```diff
@@ -17,9 +18,10 @@
 
             @if (isset($field['model']))
                 @foreach ($field['model']::all() as $connected_entity_entry)
-                    <option value="{{ $connected_entity_entry->getKey() }}"
+                    <?php $value = $owner_key === null ? $connected_entity_entry->getKey() : $connected_entity_entry->{$owner_key} ?>
```

Comment:
Consider `is_null($owner_key)` maybe? I get antsy about a comparison to `null` itself.